code/modules/species/species.dm

```diff
@@ -342,18 +343,24 @@ The slots that you can use are found in items_clothing.dm and are the inventory
 			has_limbs[ltag] = list("path" = override_limb_types[ltag])
 
 	//Build organ descriptors
+	LAZYINITLIST(limb_mapping)
 	for(var/limb_type in has_limbs)
 		var/list/organ_data = has_limbs[limb_type]
 		var/obj/item/organ/limb_path = organ_data["path"]
 		organ_data["descriptor"] = initial(limb_path.name)
+		LAZYDISTINCTADD(limb_mapping[limb_type], limb_type)
```

Comment:
Did you mean the second argument to the macro to be `organ_data`? I'm a bit lost here.

Reply:
This block is used to make sure a given tag maps to itself but I suppose I could make the default behavior to not try to map it at all. Might be clearer.

Reply:
Just noticed that the default behavior is to do this, so I will just remove the list setting from species init entirely.